src/main/java/uk/gov/hmcts/reform/professionalapi/resolver/HeaderVersionArgumentResolver.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.professionalapi.resolver;
+
+import javax.servlet.http.HttpServletRequest;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.MethodParameter;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+import uk.gov.hmcts.reform.auth.checker.spring.serviceanduser.ServiceAndUserDetails;
+
+
+@Slf4j
+public class HeaderVersionArgumentResolver implements HandlerMethodArgumentResolver {
+
+    @Override
+    public boolean supportsParameter(MethodParameter methodParameter) {
+
+        return methodParameter.getParameterAnnotation(OrgId.class) != null;

```

Comment:
switch the order to null != methodParameter.getParameterAnnotation(OrgId.class) to avoid potential nullPointer exception

Reply:
done